### PR TITLE
Czech translation

### PR DESCRIPTION
**Pull request by InnilToo**
Whoop whoop, translated to Czech. Hopefully even grammatically correct.

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -37,6 +37,12 @@
             "name": "English",
             "path": "lang/en.json",
             "flags": {}
+        },
+        {
+            "lang": "cs",
+            "name": "Česky",
+            "path": "lang/cs.json"
```
Comment (krbz999) [inline module.json:44]: Each line of an object in a json will need a trailing comma or the module will be unable to load.

(Unless it is the last line in the object.)

**Review by krbz999 — CHANGES_REQUESTED** [review]
Amend comma in the json, and ensure the module is able to load with Czech translation enabled.

File: module.json (hunk 1 above)
Comment (InnilToo) [reply module.json]: Fixxed and found parts that cannot be translated -> https://github.com/krbz999/visual-active-effects/issues/7

**Comment by InnilToo** [comment]
Fixxed and tested.

File: lang/cs.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            "EXTRA_DAYS_PER_YEAR": {
+                "NAME": "Navíc dnů v roce",
+                "HINT": "Počet dnů navíc v roce."
+            }
+        },
+        "LABELS": {
+            "TEMPORARY": "Dočasné",
+            "DISABLED": "Zakázané",
+            "DETAILS": "Detaily"
+        },
```
Comment (krbz999) [inline lang/cs.json:29]: Missing localization was fixed. This can be added here (see the en.json).

**Review by krbz999 — CHANGES_REQUESTED** [review]
Adding the one missing localization string should do it. Then I'm prepared to merge.

File: lang/cs.json (hunk 2 above)
Comment (InnilToo) [reply lang/cs.json:30]: Added "PASSIVE"

**Review by krbz999 — APPROVED** [review]

**Pull request merged**